modules/eks-aws/outputs.tf

```diff
@@ -29,3 +29,7 @@ output "kubernetes_cluster_ca_certificate" {
 output "kubernetes_token" {
   value = local.kubernetes_token
 }
+
+output "kubeconfig" {
```

Comment:
Could you please add the same output in the 2 other flavors (k3s-docker and k3os-libvirt)?